src/distribution/logistic.rs

```diff
@@ -0,0 +1,206 @@
+use distribution;
+use source::Source;
+
+/// A logistic distribution.
+#[derive(Clone, Copy)]
+pub struct Logistic {
+    mu: f64,
+    s: f64,
+}
+
+impl Logistic {
+    /// Create a logistic distribution with location `mu` and scale `s`.
+    ///
+    /// It should hold `s > 0`.
+    #[inline]
+    pub fn new(mu: f64, s: f64) -> Self {
+        should!(s > 0.0);
+        Logistic { mu: mu, s: s }
+    }
+
+    /// Return the location parameter.
+    #[inline(always)]
+    pub fn mu(&self) -> f64 { self.mu }
+
+    /// Return the scale parameter.
+    #[inline(always)]
+    pub fn s(&self) -> f64 { self.s }
+}
+
+impl Default for Logistic {
+    #[inline]
+    fn default() -> Self {
+        Logistic::new(0.0, 1.0)
```

Comment:
It's bell-curved too and really close to gaussian, but different. Wiki have good explanation about it: 

> Indeed, the logistic and normal distributions have a quite similar shape. However, the logistic distribution has heavier tails, which often increases the robustness of analyses based on it compared with using the normal distribution.

It's a reason why it often used with risk simulations.